test/functional/sqlsrv/skipif_version_not_2k14.inc

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
the name is misleading this will return true for 2014 and above.

Reply:
skipif_version_less_than_2k14.inc? Name starting to get a bit unwieldy...